### PR TITLE
Fixes and features

### PR DESCRIPTION
**Pull request by benja**
- Added DeadToggle helper component
- Added Affix support to Combobox
- Combobox no longer submits forms when selecting with Enter key
- Button now has `target` and `rel` attributes.
- Fixed old "Toggle in card" examples to use Clickable and DeadToggle

### CODE REVIEW

File: packages/box/src/clickable.tsx [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
+
+  return radio || checkbox ? (
+    <ToggleItem
+      labelClassName={props.labelClassName}
+      className="absolute inset-0 h-full w-full appearance-none cursor-pointer focus-ring"
+      type={type}
+      controlled={false}
+      onChange={() => undefined}
+      value={value}
+      name={`dead:toggle`}
```
Comment (pearofducks) [inline packages/box/src/clickable.tsx:23]: I'm pretty sure `name` should be set by userland - unless this isn't `input[name="dead:toggle"]`

Comment (benja) [reply packages/box/src/clickable.tsx:23]: Does the name even matter in this context, since the Card is handling the click? 🤔

Comment (pearofducks) [reply packages/box/src/clickable.tsx:23]: Not sure about clicks, especially not in React! But my guess is something in accessibility land cares about this. :D

Comment (benja) [reply packages/box/src/clickable.tsx]: Added an auto-generated `id` now just in case!

**Pull request merged**